invisible_cities/cities/components_test.py

```diff
@@ -152,6 +152,36 @@ def dummy_city( files_in    : Union[str, list]
     assert sorted(result) == sorted(expected)
 
 
+@mark.parametrize("order", ((0,1), (1,0)))
+def test_city_keeps_input_file_ordering(ICDATADIR, config_tmpdir, order):
+    files_in = [ os.path.join(ICDATADIR, "electrons_511keV_z250_RWF.h5")
+               , os.path.join(ICDATADIR, "electrons_1250keV_z250_RWF.h5") ]
+    files_in = [files_in[i] for i in order]
```

Comment:
Just so I understand exactly how this works, `order` sets it such that `files_in` converted into a list of two lists, with the order of the files flipped. Is that correct?

Reply:
Not a list of two lists, but a list of strings. This produces two tests. In one of them, `order == (0,1)`, and keep the original order. In the other one `order == (1,0)` and the order is reversed. In both cases, `files_in` is a list with two strings.

Reply:
Sorry yes, I was thinking of each string as a list (of characters). Okay that makes sense, thanks!